nodeletctl/cmd/root.go

```diff
@@ -0,0 +1,141 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"path"
+
+	homedir "github.com/mitchellh/go-homedir"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+	"gopkg.in/natefinch/lumberjack.v2"
+)
+
+var cfgFile string
+var logDir string
+var debugToConsole bool
+
+// hmm how can I avoid global variable
+var JSONOuput bool
+
+var (
+	clusterBootstrapFile = "/opt/pf9/airctl/conf/nodeletCluster.yaml"
+)
+
+// RootCmd represents the base command when called without any subcommands
+var RootCmd = &cobra.Command{
+	Use:     "nodeletctl",
+	Short:   "nodeletctl is a cluster manager for nodelets",
+	Long:    `nodeletctl is a cluster manager to deploy and configure nodelets on remote machines`,
+}
+
+// Execute adds all child commands to the root command and sets flags appropriately.
+// This is called by main.main(). It only needs to happen once to the rootCmd.
+func Execute() {
+	if err := RootCmd.Execute(); err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+}
+
+func init() {
+
+	cobra.OnInitialize(initConfig)
+
+	// Here you will define your flags and configuration settings.
+	// Cobra supports persistent flags, which, if defined here,
+	// will be global for your application.
+
+	RootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/airctl-config.yaml)")
```

Comment:
airctl-config ? shouldn't that be nodelet-config ?

Reply:
Fixed